modules/stanford_profile_styles/package-lock.json

```diff
@@ -1,13279 +0,0 @@
-{
```

Comment:
This file should stay in the repo and should not be deleted. It should only change if something was updated intentionally.